sdk/storage/azure-storage-file/src/main/java/com/azure/storage/file/FileAsyncClient.java

```diff
@@ -830,17 +830,17 @@ private void channelCleanUp(AsynchronousFileChannel channel) {
      * @param destinationOffset Starting point of the upload range on the destination.
      * @param sourceOffset Starting point of the upload range on the source.
      * @param sourceURI Specifies the URL of the source file.
-     * @return A response containing the {@link FileUploadRangeFromURLInfo file upload range from url info} with headers
+     * @return A response containing the {@link FileUploadRangeFromUrlInfo file upload range from url info} with headers
      * and response status code.
      */
     // TODO: (gapra) Fix put range from URL link. Service docs have not been updated to show this API
-    public Mono<Response<FileUploadRangeFromURLInfo>> uploadRangeFromURLWithResponse(long length,
-        long destinationOffset, long sourceOffset, URI sourceURI) {
+    public Mono<Response<FileUploadRangeFromUrlInfo>> uploadRangeFromURLWithResponse(long length,
```

Comment:
Are these method names also supposed to change?

Reply:
Good catch. Will make a change.